ivy/functional/frontends/paddle/tensor/creation.py

```diff
@@ -144,6 +144,15 @@ def ones_like(x, /, *, dtype=None, name=None):
     return ivy.ones_like(x, dtype=dtype)
 
 
+# []-22965
+@with_supported_dtypes("paddle")
```

Comment:
Your with_supported_dtypes implementation is not correct, this is causing the tests to fail. You have to check the paddle framework version and the crossponding dtypes supported for this function.